customer-chat/public/chat.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Chat with us</title>
+</head>
+
+<body>
+    <div id="fb-root"></div>
+    <script>(function (d, s, id) {
```

Comment:
@ChrisChinchilla  This script and the customer chat div shouldn't be in the starter since adding them is step 4 in the tutorial. Probably should just put in code comments where they are to be inserted, e.g. <!-- INSERT SDK INCLUDE SCRIPT HERE -->